x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/unmute.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { UserAtSpaceScenarios } from '../../scenarios';
+import { getUrlPrefix, getTestAlertData, ObjectRemover } from '../../../common/lib';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function createUnmuteAlertTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  describe('unmute', () => {
```

Comment:
Maybe also test that the scenario mute instance, mute alert, umute instance does what we expect (unmutes all)?

Reply:
Test added in `alerts.ts` in commit: https://github.com/elastic/kibana/pull/43712/commits/5c147c1a74f36963170504579d67cded32b5fab1.